libraries/plugins/apis/chain_api/include/steemit/plugins/chain_api/chain_api.hpp

```diff
@@ -9,7 +9,12 @@ namespace steemit { namespace plugins { namespace chain {
 
 namespace detail { class chain_api_impl; }
 
-typedef steemit::chain::signed_block push_block_args;
+struct push_block_args
+{
+   steemit::chain::signed_block block;
+   bool                         currently_syncing;
```

Comment:
Default initialize this field to false. It is used to determine what log level the chain plugin should use when pushing the block.